### PR TITLE
Add base pay to people, make base pay and durations reader-friendly

### PR DESCRIPTION
**Pull request by ian-from-dover**
Let the pictures do the talking :)  

Screenshot 2022-04-06 at 8 35 55 PM
Rates are now displayed in the form $XX to $XX.XX depending on whether 2dp is needed or not. It is rounded up.

Screenshot 2022-04-06 at 9 05 39 PM
Durations are now displayed in the form XXhXXm, rounded up. If the minutes component is not needed, minutes are not displayed.

### CODE REVIEW

File: src/main/resources/view/JobListCard.fxml [hunk 1]
```diff
@@ -12,7 +12,7 @@
       <Label fx:id="idx" minWidth="17.0" prefHeight="17.0" prefWidth="17.0" text="1" />
       <Label fx:id="desc" layoutX="35.0" layoutY="27.0" maxWidth="100.0" minWidth="100.0" prefWidth="100.0" text="Fix washing machine" wrapText="true" />
       <Label fx:id="duration" layoutX="265.0" layoutY="27.0" maxWidth="60.0" minWidth="60.0" prefWidth="60.0" text="PT48H" wrapText="true" />
-      <StackPane alignment="CENTER_LEFT" maxWidth="80.0" minWidth="80.0" prefWidth="80.0">
+      <StackPane alignment="CENTER_LEFT" maxWidth="30.0" minWidth="30.0" prefWidth="30.0">
```
Comment (ovidharshini) [inline src/main/resources/view/JobListCard.fxml:15]: Perhaps something to consider in future revisions might be to make use of XSLT?
Something like 
`<xsl:param name='count'>28</xsl:param>` 
`<param name='namelist' split=',' max-len='100' max-occurrences='{$count}'/>`
(taken from [source](https://stackoverflow.com/questions/49993748/declaring-and-using-a-variable-in-xml))
might make edits more convenient in general, though the files for this project are currently small enough that this probably does not add much value at the moment.

File: src/main/java/peoplesoft/ui/regions/JobCard.java [hunk 2]
```diff
@@ -51,10 +51,17 @@
...
+        int hH = job.getDuration().toHoursPart();
+        int mins = job.getDuration().toMinutesPart();
+        String mM = mins == 0 ? "" : (mins + "m");
+
+        // assign values
         this.job = job;
         idx.setText(displayedIndex + "");
         desc.setText(job.getDesc());
-        duration.setText(job.getDuration().toString());
+        duration.setText(String.format("%dh%s", hH, mM));
```
Comment (ovidharshini) [inline src/main/java/peoplesoft/ui/regions/JobCard.java:64]: Maybe adding a space or a colon as a visual separator between the hours and minutes would make it more readable?

**Review by ovidharshini — COMMENTED** [review]
looks good!

File: src/main/resources/view/PersonListCard.fxml [hunk 3]
```diff
@@ -8,8 +8,9 @@
    <children>
       <Label fx:id="idx" maxHeight="17.0" maxWidth="17.0" minHeight="17.0" minWidth="17.0" prefHeight="17.0" prefWidth="17.0" text="1" />
       <Label fx:id="name" layoutX="35.0" layoutY="27.0" maxWidth="60.0" minWidth="60.0" prefWidth="60.0" text="Nicole Lee Siying Rajara" wrapText="true" />
-      <Label fx:id="amtDue" layoutX="202.0" layoutY="37.0" maxWidth="60.0" minWidth="60.0" prefWidth="60.0" text="\$8888.88" wrapText="true" />
+      <Label fx:id="amtDue" layoutX="202.0" layoutY="37.0" maxWidth="100.0" minWidth="100.0" prefWidth="100.0" text="\$8888.88" wrapText="true" />
       <FlowPane fx:id="tags" alignment="CENTER_LEFT" maxWidth="65.0" minWidth="65.0" prefWidth="65.0" prefWrapLength="50.0" />
+      <Label fx:id="basePay" layoutX="302.0" layoutY="37.0" maxWidth="60.0" minWidth="60.0" prefWidth="60.0" text="\$30 / H" wrapText="true" />
```
Comment (zhongfu) [inline src/main/resources/view/PersonListCard.fxml:13]: it's `$30/h` now, right?

**Review by zhongfu — APPROVED** [review]

File: src/main/resources/view/JobListCard.fxml (hunk 1 above)
Comment (ian-from-dover) [reply src/main/resources/view/JobListCard.fxml:15]: I wish I knew about this 😭

Comment (ian-from-dover) [reply src/main/resources/view/JobListCard.fxml:15]: But i'm not sure if SceneBuilder will overwrite it when I make changes in that interface, so i'll invest my time in fixing other bugs. Thanks for the good suggestion.

File: src/main/java/peoplesoft/ui/regions/JobCard.java (hunk 2 above)
Comment (ian-from-dover) [reply src/main/java/peoplesoft/ui/regions/JobCard.java:64]: Good suggestion

**Pull request merged**